docs/en/week13/13-2.md

```diff
@@ -92,7 +92,7 @@ ChebNets are GCNs that can be used for any arbitrary graph domain, but the limit
 We can extend ChebNets to multiple graphs using a 2D spectral filter. This may be useful, for example, in recommender systems where we have movie graphs and user graphs. Multi-graph ChebNets have the activation equation as below.
 
 $$
-h^{l+1}=\eta(\hat{w}(\Delta_1,\Delta_2)*h^l)
+h^{l+1}=\eta(\hat{w}(\Delta_1,\Delta_2)h^l)
```

Comment:
Didn't get this. What's wrong?

<img width="726" alt="Screenshot 2020-05-15 at 21 48 33" src="https://user-images.githubusercontent.com/2119355/82107469-f7658180-96f5-11ea-9dce-78409c2032b8.png">

Reply:
Hi, in my text editor it seemed like it was interpreted as both * the math symbol in LateX and * the italic symbol in markdown. I therefore thought it might be this causing what you fixed in f568d60. 
This has nothing to do with the math itself so never mind!

Reply:
I have reverted the commit